src/unsafe/simplify.ts

```diff
@@ -50,8 +50,8 @@ import { simplifyXor } from './xor'
 export const unsafeSimplify = (
```

Comment:
I don't know if it's worth maintaining unsafeSimplify, as not going to use it, I don't think.